cli/server.js

```diff
@@ -0,0 +1,44 @@
+require('zone.js/dist/zone-node');
+const { Request, Response } = require('express');
+const { platformServer, renderModuleFactory } = require('@angular/platform-server');
+const { ngExpressEngine } = require('@nguniversal/express-engine');
+// Import module map for lazy loading
+const { provideModuleMap } = require('@nguniversal/module-map-ngfactory-loader');
+
+const express = require('express');
+const fs = require('fs');
+
+
+const files = fs.readdirSync(`${process.cwd()}/dist-server`);
+const mainFiles = files.filter(file => file.startsWith('main'));
+const hash = mainFiles[0].split('.')[1];``
```

Comment:
What are `` at the end of the line?

Reply:
just a typo :D also forgot to update that with the one have for prerender.js